api/src/main/resources/messages.properties

```diff
@@ -31,7 +31,7 @@ registrationapp.patient.gender.question=What's the patient's gender?
 
 registrationapp.patient.contactInfo.label=Contact Info
 
-registrationapp.patient.address.question=What is the patient's address?
+registrationapp.patient.address.question=What's the patient's address?
```

Comment:
@druchniewicz   thanks for taking on this,do we need this line?

Reply:
I noticed that everywhere in messages.properties is used shorter form, for example: 
* registrationapp.patient.name.question=What's the patient's name?
* registrationapp.patient.phone.question=What's the patient phone number?
* registrationapp.patient.birthdate.question=What's the patient's birth date?
* registrationapp.patient.gender.question=What's the patient's gender?
and only in patient address is used: 
registrationapp.patient.address.question=What is the patient's address?
so I changed it to be a consistent with all properties. What do you think?
This is a tiny change and I can change it, but in my opinion it will be more consistent. Let me know, what do you think about it

Reply:
@druchniewicz ok,i think the we got to be more consistent,the change is ok

Reply:
Hmm... I totally agree with consistency, but, unfortunately in this case, changing the text will mean clearing out all the current translations of this string, not sure it's worth having to make sure someone re-translates all of them?

Reply:
ah, ok, I understand, so you're right, I will restore the old version value of this message property